template/_package.json

```diff
@@ -28,7 +28,11 @@
     <%_ if (eslint) { _%>
     "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
     <%_ } _%>
-    <%_ if (test !== 'none') { _%>
+    <%_ if (test === 'ava') { _%>
+    "test": "<%= test %>",
+    "test:unit": "cross-env TEST=unit ava --config unit.config.js",
+    "test:e2e": "cross-env TEST=e2e ava --config e2e-tests.config.js",
```

Comment:
`e2e-tests.config.js` or `e2e.config.js` ?

BTW, how do you think moving test related config files to `test` dir ?

Reply:
Yep. It could also be flexible enough to make use of all types of testing provided by Ava.

I'll update the PR and let's get some feedback from others too?